Tests/BootCamp.Chapter.Tests/PokerComboTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using BootCamp.Chapter.Gambling.Poker;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class PokerComboTests
+    {
+        [Fact]
+        public void Smaller__Given_Set1_Is_Smaller_Set2_Returns_True()
+        {
+            var set1 = new PokerCombo(PokerCombo.Sets.Flush, 1);
+            var set2 = new PokerCombo(PokerCombo.Sets.RoyalFlush, 12);
+
+            var actual = set1 < set2;
+
+            Assert.True(actual);
+        }
+
+        [Fact]
+        public void Smaller__Given_Set1_Is_Equal_Set2_But_Set1_Lower_Score_Returns_True()
+        {
+            var set1 = new PokerCombo(PokerCombo.Sets.RoyalFlush, 1);
+            var set2 = new PokerCombo(PokerCombo.Sets.RoyalFlush, 12);
+
+            var actual = set1 < set2;
+
+            Assert.True(actual);
+        }
+
+        [Fact]
+        public void Smaller__Given_Set1_Is_Greater_Set2_Returns_False()
+        {
+            var set1 = new PokerCombo(PokerCombo.Sets.RoyalFlush, 1);
+            var set2 = new PokerCombo(PokerCombo.Sets.Flush, 12);
+
+            var actual = set1 < set2;
+
+            Assert.False(actual);
+        }
+
+        [Fact]
```

Comment:
use theory

Reply:
refractored the test to use theory  so now 3 seperate tests for equal, smaller and greater then